src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -362,6 +362,12 @@ private static void GetLogoFile(this CanvasDocument app)
             }
         }
 
+        // Stores an .fx.yaml file and it's directory bool value
+        public struct fxYamlGroup{
+            public Microsoft.PowerPlatform.Formulas.Tools.DirectoryReader.Entry file;
```

Comment:
Does this need to be a struct, or should it just be a tuple used local to this function?

i.e. `List<(Entry file, bool canSearch)>`